code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -380,11 +380,17 @@
 	return TRUE
 
 /datum/dynamic_ruleset/roundstart/revs/execute(forced = FALSE)
+	var/list/all_heads = SSjob.get_all_heads()
+	var/list/all_security = SSjob.get_all_sec()
+
+	if (all_heads.len < 2  || all_security.len < 2 || all_heads.len + all_security.len < 5)
```

Comment:
this really shouldn't go in execute(), it's not supposed to exit if it doesn't have to. Put it in ready() instead.